Source/relay/TourGuide/Items of the Month/2021/Daylight Shavings Helmet.ash

```diff
@@ -1,93 +1,90 @@
 //Daylight Shavings Helmet
+
+string [int, string] dshBuffOrder = {
+	{"Spectacle": "+50% item and +5 " + HTMLGenerateSpanFont("Spooky", "grey") + " resist"},
+	{"Toiletbrush": "+25 ML and +5 " + HTMLGenerateSpanFont("Stench", "green") + " resist"},
+	{"Barbell": "+25 Muscle and +50% Gear drops"},
+	{"Grizzly": "+25-50 MP regen and +5 " + HTMLGenerateSpanFont("Cold", "blue") + " resist"},
+	{"Surrealist": "+25 Mysticality and +50% Food drops"},
+	{"Musician": "+25 Moxie and +50% Booze drops"},
+	{"Gull-Wing": "+100% init and +5 " + HTMLGenerateSpanFont("Hot", "red") + " resist"},
+	{"Warlord": "+25 Weapon damage and +10% Crit"},
+	{"Wizard": "+25 Spell damage and +10% Spell Crit"},
+	{"Cowboy": "+25 Ranged damage and +50 maximum HP"},
+	{"Friendly": "+100% meat and +5 " + HTMLGenerateSpanFont("Sleaze", "purple") + " resist"}
+};
+
+string [string] getNthClassBuff(int n) {
+	int classInt = my_class().to_int();
+	int buffIndex;
+	if (classInt < 7)
+	{
+		buffIndex = ((classInt * n) % 11);
+	}
+	else if (classInt > 6)
+	{
+		buffIndex = ((((classInt % 6 + 1) % 11)));
```

Comment:
hrm wasn't there a thing where boris had some weird exception handling? or is this the actual fix... i haven't actually worried about DSH much in avatar paths that didn't have pins telling me exactly what the dsh alignment was lmao

Reply:
Good question. I didn't change anything about the avatar path logic that was already there, but the formula here matches up with what was in the spading spreadsheet, so I believe it should work.